.github/workflows/integration-tests.yml

```diff
@@ -59,62 +59,63 @@ jobs:
           name: build-reports
           path: mobile/build/reports/
 
-  test-iosched:
-    name: iosched api ${{ matrix.api-level }} sentry enabled ${{ matrix.enable-sentry }}
-    runs-on: macos-latest # enables hardware acceleration in the virtual machine
-    timeout-minutes: 45
-    strategy:
-      fail-fast: false
-      matrix:
-        api-level: [23, 26]
-        # To find out if Sentry causes tests to fail or maybe the tests itself are flaky
-        # we run them with and without Sentry enabled.
-        enable-sentry: [true, false]
-
-    steps:
-      - uses: actions/checkout@v2
-        with:
-          repository: 'google/iosched'
-          ref: '6f96c1528c00d3d89bedb33e5a0a10d67629e8bd'
-
-      - uses: actions/checkout@v2
-        with:
-          path: 'sentry-android-gradle-plugin'
-
-      - name: Add Sentry
-        if: ${{ matrix.enable-sentry == true }}
-        run: git apply ./sentry-android-gradle-plugin/scripts/add-sentry-to-iosched.patch
-
-      - name: Copy CI gradle.properties
-        run: mkdir -p ~/.gradle ; cp .github/ci-gradle.properties ~/.gradle/gradle.properties
-
-      - name: Set up JDK 11
-        uses: actions/setup-java@v1
-        with:
-          java-version: 11
-
-      - name: Run instrumentation tests
-        uses: reactivecircus/android-emulator-runner@48744f20730dfa6545f371626eb9a2c3908199b0
-        with:
-          api-level: ${{ matrix.api-level }}
-          arch: x86
-          disable-animations: true
-          script: |
-            touch emulator.log                    # create log file
-            chmod 777 emulator.log                # allow writing to log file
-            adb logcat >> emulator.log &          # pipe all logcat messages into log file as a background process
-            ./gradlew mobile:cAT --stacktrace         # here run your tests
-
-      - name: Upload device logs
-        if: always()
-        uses: actions/upload-artifact@v2
-        with:
-          name: emulator-logs
-          path: emulator.log
-
-      - name: Upload test reports
-        if: always()
-        uses: actions/upload-artifact@v2
-        with:
-          name: test-reports
-          path: mobile/build/reports/
+# https://github.com/getsentry/sentry-android-gradle-plugin/issues/271 we'll switch to testing the duckduckgo app
```

Comment:
Can we make a separate PR for that? so we merge into the main branch and merge it back to this PR.